packages/backend/src/open_payments/payment/outgoing/model.ts

```diff
@@ -120,6 +120,7 @@ export class OutgoingPayment
     const data: PaymentData = {
       id: this.id,
       walletAddressId: this.walletAddressId,
+      client: this.client,
```

Comment:
I think `client` should actually be added to the `OutgoingPayment` class. It looks like that is what you have in mind here?

Reply:
The `PaymentData` from `packages/backend/src/open_payments/payment/outgoing/model.ts` is actually `OutgoingPaymentResponse` but with the `Omit`, snippet:
```
export type PaymentData = Omit<OutgoingPaymentResponse, 'failed'> & {
```
I added the `client?: string` property to the `OutgoingPaymentResponse` . 

Furthermore the `OutgoingPaymentEvent` makes use of the `PaymentData` as the payload "data". I am thinking maybe we should rename the `PaymentData` to perhaps `OutgoingPaymentData`?
```typescript
export class OutgoingPaymentEvent extends WebhookEvent {
  public type!: OutgoingPaymentEventType
  public data!: PaymentData
```

Reply:
Never mind, I thought client was not part of the model yet, but it is.

Reply:
@koekiebox still one here ^